### DISCUSSION TITLE
GraphQL matching of project board name unexpected

### DISCUSSION THREAD

**Discussion by MarkoSchuetz** [community/community#112979] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I have two project boards one, say, called "XYZ Development" another called "XYZ Project". Using organization->projectsV2->query with "XYZ Project" and `first:1` I was expecting to get the project board that matches exactly. Instead, I got the one that matches only partially. This is unexpected and I did not find any mention about whether or not the match should be exact, prefix,... Can anyone help me out here?

**Comment by Devineonline99** [comment]
I am sorry is this for the bugs that are on my phones right well if I could have some guidance I did the best I could but  like said when I made my first discovery I am not a specialist when it comes to bug reports or virus or even interested in catching or solving them I was forced to try and fix it because I need my phones to work it affected my whole accounts to business to my bots to my future works everything my surface computer. from my surface from my other phones I can't make a proper report I am on a shifty Samsung a51u and I am so tired of removing this crop of my phones I noticed the files that are up  on Google under maps was showing the file google was sending me to get into My accounts this phone has had numerous of breakdowns and then an update or a backup will come back to me and there it is again I am locked out of my bu a business accounts and everything  is gone back to crap .if I could have someone one guide me thru this by giving access I am willing to do that but I can't get anywhere especially when I keep getting these updates that just put the same old google chrome that thinks I am talking to you from my chrome book this is  an andriod phone

**Reply by Devineonline99** [reply]
I am very tired I have been at this for weeks now and I just wanted to get back into my rightful accounts get my bot under control and get my business and my finances under control instead I had to do this big long process of just getting my phones to work with my computer get my microsoft 365 under control and not get involved in solving bugs and leaks I will leave this st that for now until I can get my accounts back because I am not going to be filling all this information out over and over again .thank you I was going to do the whole report but not if my phone keeps doing this

**Reply by AE31231** [reply]
- [ ] AE T
